.rubocop_todo.yml

```diff
@@ -23,7 +23,7 @@ RSpec/ExpectActual:
     - 'spec/routing/**/*'
     - 'spec/controllers/catalog_controller_spec.rb'
 
-# Offense count: 2
+# Offense count: 3
```

Comment:
I don't see any new specs, so how could their be a new violation?

Reply:
probably it passed with this exception but todo file not updated

Reply:
I didn't think that was possible. Like, it should have failed rubocop's inspection before a merge 🤷‍♂️